runtime/onert/core/src/exec/train/TrainableExecutor.cc

```diff
@@ -136,7 +136,7 @@ void TrainableExecutor::forwardImpl(const ExecutionObservee &subject, bool train
       ruy::profiler::ScopeLabel label(code.op->name());
 #endif
       auto &tn_seq = code.tn_seq;
-      tn_seq->forward(training);
+      tn_seq->forward(training && code.op->isRequiredForBackward());
```

Comment:
It is a missing equivalent from tracing path:
https://github.com/Samsung/ONE/pull/13251/files#diff-c3aea1c2d60b52608ea27f07b39c07268fd1e4d9fbc88fe6c3517318fa713cf3R123